### PR TITLE
DDF-3929 Remove Standard Search UI from installer

### PR DESCRIPTION
**Pull request by austinsteffes**
#### What does this PR do?
Removes search-ui from installer to decrease start time.
#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
@ahoffer 
@garrettfreibott 
@vinamartin 
@peterhuffer 
#### Select relevant component teams: 
@codice/ui 
#### Ask 2 committers to review/merge the PR and tag them here. If you don't know who to ask, you can request reviews in https://groups.google.com/forum/#!forum/ddf-developers .
(please choose ONLY two committers from below, delete the rest)
@tbatie 
#### How should this be tested? (List steps with links to updated documentation)
do a build of the system. search-ui shouldn't start up in the installer. 
#### What are the relevant tickets?
[DDF-3929](https://codice.atlassian.net/browse/DDF-3929)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by austinsteffes** [comment]
build now

**Comment by austinsteffes** [comment]
Split this PR into just removing search-ui from the installer and updated documentation based on @peterhuffer advice. build now

File: catalog/ui/search-ui/standard/src/main/doc/_contents/help-contents.adoc [hunk 1]
```diff
@@ -38,6 +39,15 @@ Additionally, a user can save individual records that were returned by the query
 they can be referenced in the future. The user can also save the search criteria to a workspace so
 the query can be run again.
 
+==== Turning on Search UI
```
Comment (vinamartin) [inline catalog/ui/search-ui/standard/src/main/doc/_contents/help-contents.adoc]: ✏️ I think you'll want to use the same verbiage as we use in our other installation sections, which I think might be "Installing ...."

File: catalog/ui/search-ui/standard/src/main/doc/_contents/help-contents.adoc [hunk 2]
```diff
@@ -38,6 +39,15 @@ Additionally, a user can save individual records that were returned by the query
...
+
+The Search UI is turned off by default, but can be turned on through the Admin UI.
```
Comment (vinamartin) [inline catalog/ui/search-ui/standard/src/main/doc/_contents/help-contents.adoc]: ✏️ Similarly, "uninstalled", but I'd double check with our other docs sections.

Comment (austinsteffes) [reply catalog/ui/search-ui/standard/src/main/doc/_contents/help-contents.adoc]: fixed. thanks

File: catalog/ui/search-ui/standard/src/main/doc/_contents/help-contents.adoc (hunk 1 above)
Comment (austinsteffes) [reply catalog/ui/search-ui/standard/src/main/doc/_contents/help-contents.adoc]: fixed

**Comment by austinsteffes** [comment]
finalized stuff. build now

**Comment by austinsteffes** [comment]
build now

**Comment by andrewkfiedler** [comment]
Feels like we should mention that this is for the Standard Search UI and that it’s deprecated.

**Comment by shaundmorris** [comment]
Okay good point @andrewkfiedler. I think we can update the feature name (like `search-ui-DEPRECATED`) to make it clear as well as the docs that this is for the Standard Search UI.

**Comment by austinsteffes** [comment]
build now

**Comment by shaundmorris** [comment]
build now

**Comment by shaundmorris** [comment]
❗️ @austinsteffes we will need to change this line in the integration tests: https://github.com/codice/ddf/blob/master/distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java#L393 I think we should remove "search-ui" from there rather than replace it.

**Comment by austinsteffes** [comment]
build now

**Comment by shaundmorris** [comment]
build now

**Comment by austinsteffes** [comment]
Changed deprecation to new implementation.
Build now

**Comment by austinsteffes** [comment]
connection reset. build now

**Comment by austinsteffes** [comment]
Adding search-ui(now search-ui-deprecated) back into itest.
Poised in future to remove it from there but will be a separate ticket. 
Build now.

**Comment by austinsteffes** [comment]
passed locally. build now

**Comment by austinsteffes** [comment]
rebase build now

**Comment by mcalcote** [comment]
From the most recent failure

One or more dependencies were identified with known vulnerabilities in DDF :: Platform :: Solr :: Factory:

```
curator-recipes-2.8.0.jar (org.apache.curator:curator-recipes:2.8.0, cpe:/a:apache:zookeeper:2.8.0) : CVE-2018-8012
hadoop-auth-2.7.4.jar (org.apache.hadoop:hadoop-auth:2.7.4, cpe:/a:apache:hadoop:2.7.4) : CVE-2017-15718
zookeeper-3.4.10.jar (org.apache.zookeeper:zookeeper:3.4.10, cpe:/a:apache:zookeeper:3.4.10) : CVE-2018-8012
spring-core-4.3.10.RELEASE.jar (org.springframework:spring-core:4.3.10.RELEASE, cpe:/a:pivotal_software:spring_framework:4.3.10, cpe:/a:pivotal:spring_framework:4.3.10) : CVE-2018-1257, CVE-2018-1258, CVE-2018-1275, CVE-2018-1199, CVE-2018-1271, CVE-2018-1272
solr-dependencies-2.13.0-SNAPSHOT.jar/META-INF/maven/org.apache.hadoop/hadoop-hdfs/pom.xml (org.apache.hadoop:hadoop-hdfs:2.7.4, cpe:/a:apache:hadoop:2.7.4) : CVE-2017-15718
```

**Comment by clockard** [comment]
Bypassed ci/sonarqube. CI only failed for owasp which this pr doesn't have anything to do with.

**Pull request merged**